packs/feats/scouts-charge.json

```diff
@@ -11,7 +11,7 @@
         },
         "category": "class",
         "description": {
-            "value": "<p>You meander around unpredictably, and then ambush your opponents without warning.</p>\n<p>Choose one enemy. Stride, @UUID[Compendium.pf2e.actionspf2e.Item.Feint] against that opponent, and then make a Strike against it. For your Feint, you can attempt a Stealth check instead of the Deception check that's usually required, using the terrain around you to surprise your foe.</p>"
+            "value": "<p>You meander around unpredictably, and then ambush your opponents without warning.</p>\n<p>Choose one enemy. Stride, [[/act feint skill=stealth]]{Feint} against that opponent, and then make a Strike against it. For your Feint, you can attempt a Stealth check instead of the Deception check that's usually required, using the terrain around you to surprise your foe.</p>"
```

Comment:
Would it be better to put the action on Stealth to call it out specifically, since they may choose to use Deception instead?

Reply:
That makes sense